app/javascript/src/casa_case.js

```diff
@@ -7,14 +7,19 @@ import Swal from 'sweetalert2'
 
 function addCourtMandateInput () {
   const list = '#mandates-list-container'
+  const ref = $(list).data('ref') || 'casa_case'
```

Comment:
The changes in this file allow for this to work for casa_cases as well as past_court_dates.